CONTRIBUTING.md

```diff
@@ -0,0 +1,190 @@
+
+If you haven't already, make sure to read our [README](https://github.com/GetStream/stream-chat-android/blob/main/README.md), to learn all about the repo, our codebase, documentation and much more.
+
+![Android Repository Structure](https://user-images.githubusercontent.com/17215808/173359688-31ec7680-0a59-4ad6-91a4-f538a935ce26.PNG)
+
+### Important modules & files 🧱
+
+- `stream-chat-android-client`: Holds all the code connected to the **low-level-client**, used to request data from and send information to the API.
+- `stream-chat-android-offline`: As an addition to the **client** module, you can use **offline** to introduce persistence and caching to your app.
+- `stream-chat-android-ui-common`: Common UI-related code and utility functions.
+
+- `stream-chat-android-compose-sample`: The main sample app featuring the Compose SDK. 
+- `stream-chat-android-compose`: Our Compose SDK that features channels and messages components, as well as other utilities.
+- `stream-chat-android-ui-components-sample`: The main sample app featuring the XML (UI Components) SDK. 
+- `stream-chat-android-ui-components`: Our XML (UI Components) SDK that features channels and messages components, as well as other utilities.
+
+- `CHANGELOG.md` : List of changes made in each version of the SDK.
+- `DEPRECATIONS.md`: List of deprecated code in our SDK and its deprecation process timeline.
+
+These provide corresponding artifacts on [Maven Central](https://search.maven.org/search?q=g:io.getstream) that you can plug into your projects. There are several other modules and files available in the repository and we've outlined them in our [README](https://github.com/GetStream/stream-chat-android), so be sure to check them out.
+
+### Local Setup
+
+Congratulations! 🎉.  You've successfully cloned our repo, and you are ready to make your first contribution. Our setup is super simple, so you don't have to do any additional work.
+
+As long as you have the latest stable [Android Studio](https://developer.android.com/studio) version, you should be fine. :]
+
+---
+
+# How can I contribute?
+
+Are you ready to dive into code? It's pretty easy to get up and running with your first Stream contribution. Although, you don't have to write code to contribute to our repository. There are a few ways to help:
+
+* Opening issues/bug reports.
+* Proposing feature requests and SDK improvements.
+* Creating pull requests to improve our codebase.
+
+Let's go over the process for each of these items!
+
+## Opening issues & bug reports 🐛
+
+Using the GitHub repository and project management system, you can open issues in any public repo. Before filing bugs, take a look at our existing backlog. For common items or reports, there might be an existing ticket on GitHub.
+
+To quickly narrow down the number of tickets on Github, try filtering based on the label that best suites the bug or a part of our SDK.
```

Comment:
Should we also mention our projects? Maybe sometimes it might be easier to find issues there?